.github/workflows/daily-test.yml

```diff
@@ -3,15 +3,16 @@ name: Daily Test
 on:
   schedule:
     # More information on cron https://crontab.guru/
-    # GitHub actions is using UTC time. Scheduling action at 5 am PST
-    - cron: '0 13 * * *'
+    # GitHub actions is using UTC time. Scheduling action at 3 am PST (time for 2 tries of each demo flavor).
+    - cron: '0 11 * * *'
 
   workflow_dispatch:
 
 env:
   SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
   SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  OS_VERSION_LIST: ["17", "18"]
```

Comment:
pls rerun the test for this change

Reply:
It will not work unless upgrading Appium.